heroic-core/src/test/java/com/spotify/heroic/http/tracing/TextFormatGetterTest.java

```diff
@@ -68,4 +68,14 @@ public void testNullSpan () {
     }
   }
 
+  @Test(expected = SpanContextParseException.class)
+  public void testNullRequestHeader () throws Exception {
+
+    final List<String> headers = new ArrayList<>();
+
+    doReturn(headers).when(request).getRequestHeader("X-Cloud-Trace-Context");
```

Comment:
This test is the same as the one above it (`testNullSpan`). I think the intention is to test a null with `doReturn(null).when(...`